dev/io.openliberty.microprofile.openapi.4.0.internal/bnd.bnd

```diff
@@ -0,0 +1,68 @@
+#*******************************************************************************
+# Copyright (c) 2024, 2025 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+
+Bundle-Name: MicroProfile OpenAPI 4.0 Impl
+Bundle-SymbolicName: io.openliberty.microprofile.openapi.4.0.internal
+Bundle-Description: MicroProfile OpenAPI 4.0 Impl, version ${bVersion}
+
+bVersion=1.0
+
+javac.source: 11
+javac.target: 11
+
+Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=11))"
+
+Import-Package: \
+    io.openliberty.microprofile.openapi.internal.resources,\
+    io.openliberty.microprofile.openapi.internal.resources.validation,\
+    *
+
+Export-Package: \
+    io.openliberty.microprofile.openapi40.internal.impl
```

Comment:
Usually packages with `.internal.` in the name are not exported.   Do we need to remove `.internal` from the package name?

Reply:
It's internal implementation detail which is exported because this bundle is shared between 4.0 and 4.1. Other components shouldn't be using it and it's not API or SPI.

From the [naming guidelines](https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-naming-guidelines) I think I should be using `internal` in the name for this case.